src/WSRequestHandler_SceneItems.cpp

```diff
@@ -440,154 +441,6 @@ RpcResponse WSRequestHandler::SetSceneItemRender(const RpcRequest& request) {
 	return request.success();
 }
 
-/**
-* Sets the coordinates of a specified source item.
-*
-* @param {String (optional)} `scene-name` Name of the scene the scene item belongs to. Defaults to the current scene.
-* @param {String} `item` Scene Item name.
-* @param {double} `x` X coordinate.
-* @param {double} `y` Y coordinate.
-
-*
-* @api requests
-* @name SetSceneItemPosition
-* @category scene items
-* @since 4.0.0
-* @deprecated Since 4.3.0. Prefer the use of SetSceneItemProperties.
-*/
-RpcResponse WSRequestHandler::SetSceneItemPosition(const RpcRequest& request) {
```

Comment:
Why are you removing this? 4.9 is not a breaking release, and your changes summary in the PR's description don't state that change.

Reply:
It's been deprecated since 4.3.0 so I figured I'd remove it once and for all. I can keep it in

Reply:
Let's keep it in for now.